scripts/library.sh

```diff
@@ -58,6 +58,16 @@ function warning() {
   make_banner "!" "$1"
 }
 
+# Install wrk. Used for performance benchmarking
+# libssl-dev needs to be installed before running this.
+function install_wrk() {
```

Comment:
Why not build and install the binary in the image itself, instead of building it every time you need it?

Reply:
Updated